_site/projects/index.html

```diff
@@ -124,6 +124,20 @@ <h1 class="post-title">Eklavya Projects</h1>
 <!-- Display projects without categories --><!-- Generate cards for each project -->
```

Comment:
remove this file from the commit as well (basically remove the folder of _site from your commit only)